src/cart/src/cart/crt_rpc.c

```diff
@@ -536,224 +536,222 @@ crt_req_fill_tgt_uri(struct crt_rpc_priv *rpc_priv, crt_phy_addr_t base_uri)
 	return DER_SUCCESS;
 }
 
-/* Note: Currently unused function as logic needs to be rewritten */
-int
-crt_req_uri_lookup_retry(struct crt_grp_priv *grp_priv,
-			 struct crt_rpc_priv *rpc_priv)
-{
-	struct crt_context	*crt_ctx;
-	crt_rpc_t		*rpc_pub = &rpc_priv->crp_pub;
-	int			 rc;
-
-	/* destroy previous URI_LOOKUP req, addref in crt_req_uri_lookup_psr */
-	RPC_PUB_DECREF(rpc_priv->crp_ul_req);
-	rpc_priv->crp_ul_req = NULL;
-
-	rc = crt_grp_psr_reload(grp_priv);
-	if (rc != 0) {
-		RPC_ERROR(rpc_priv,
-			  "crt_grp_psr_reload(grp %s) failed, rc: %d\n",
-			  grp_priv->gp_pub.cg_grpid, rc);
-		D_GOTO(out, rc);
-	}
-
-	crt_ctx = rpc_pub->cr_ctx;
-
-	D_MUTEX_LOCK(&crt_ctx->cc_mutex);
-	if (!crt_req_timedout(rpc_priv))
-		crt_req_timeout_untrack(rpc_priv);
-	crt_set_timeout(rpc_priv);
-	rc = crt_req_timeout_track(rpc_priv);
-	D_MUTEX_UNLOCK(&crt_ctx->cc_mutex);
-	if (rc != 0) {
-		RPC_ERROR(rpc_priv,
-			  "crt_req_timeout_track failed, rc: %d\n",
-			  rc);
-		D_GOTO(out, rc);
-	}
-
-	rpc_priv->crp_state = RPC_STATE_INITED;
-	rc = crt_req_send_internal(rpc_priv);
-
-out:
-	return rc;
-}
+static int
+crt_issue_uri_lookup(crt_context_t ctx, crt_group_t *group,
+			d_rank_t contact_rank, uint32_t contact_tag,
```

Comment:
```suggestion
		     d_rank_t contact_rank, uint32_t contact_tag,
```

Reply:
If these are current can you accept them, although checkpatch does have an issue where it only checks continued indentation on the second line, so L542 and L543 probably need changing as well.